app/ui-utils/client/index.js

```diff
@@ -6,6 +6,7 @@ export { menu } from './lib/menu';
 export { call } from './lib/callMethod';
 export { erase, hide, leave } from './lib/ChannelActions';
 export { MessageAction } from './lib/MessageAction';
+export { miniAppAtion } from './lib/miniAppAction';
```

Comment:
`miniAppAction` instead of `miniAppAtion ` ?

Reply:
Where is it being used??

Reply:
This is not used yet. I just expose it the index file to let others know this lib. Should we remove it?

Reply:
We can keep it, just fix the typo